handsontable/src/dataMap/metaManager/index.js

```diff
@@ -133,20 +133,22 @@ export default class MetaManager {
    *
    * @param {number} physicalRow The physical row index.
    * @param {number} physicalColumn The physical column index.
-   * @param {object} options Additional options that are used to extend the cell meta object.
-   * @param {number} options.visualRow The visual row index of the currently requested cell meta object.
-   * @param {number} options.visualColumn The visual column index of the currently requested cell meta object.
+   * @param {object} additionalCellMeta Additional options that are used to extend the cell meta object.
+   * @param {number} additionalCellMeta.visualRow The visual row index of the currently requested cell meta object.
+   * @param {number} additionalCellMeta.visualColumn The visual column index of the currently requested cell meta object.
+   * @param {object} options Execution options for the `getCellMeta` method.
+   * @param {boolean} [options.executeCellsFunction] If `false`, disables the execution of the `cells` function. Default: `true`.
    * @returns {object}
    */
-  getCellMeta(physicalRow, physicalColumn, { visualRow, visualColumn }) {
+  getCellMeta(physicalRow, physicalColumn, { visualRow, visualColumn }, options = { executeCellsFunction: true }) {
     const cellMeta = this.cellMeta.getMeta(physicalRow, physicalColumn);
 
     cellMeta.visualRow = visualRow;
     cellMeta.visualCol = visualColumn;
     cellMeta.row = physicalRow;
     cellMeta.col = physicalColumn;
 
-    this.runLocalHooks('afterGetCellMeta', cellMeta);
+    this.runLocalHooks('afterGetCellMeta', cellMeta, options);
```

Comment:
I'd not extend the hook with the 3th option but rather go with updating the cellMeta directly. Something like
```js
    cellMeta.visualRow = visualRow;
    cellMeta.visualCol = visualColumn;
    cellMeta.row = physicalRow;
    cellMeta.col = physicalColumn;
    cellMeta._skipMetaExtension = skipMetaExtension;

    this.runLocalHooks('afterGetCellMeta', cellMeta);
```

Reply:
After thinking about it, I think that, in this case, it is actually better not to modify the cell meta and pass the option to the hook 🤔

Reply:
ok